examples/fixtures/example.sdl.yaml

```diff
@@ -1,38 +1,58 @@
-version: '2.0'
+# Welcome to the Akash Network! 🚀☁
+# This file is called a Stack Definition Laguage (SDL)
+# SDL is a human friendly data standard for declaring deployment attributes. 
+# The SDL file is a "form" to request resources from the Network. 
+# SDL is compatible with the YAML standard and similar to Docker Compose files.
+
+---
+# Indicates version of Akash configuration file. Currently only "2.0" is accepted.
+version: "2.0"
+
+# The top-level services entry contains a map of workloads to be ran on the Akash deployment. Each key is a service name; values are a map containing the following keys:
+# https://docs.akash.network/intro-to-akash/stack-definition-language#services
```

Comment:
small detail but I updated the links in this SDL in the [cloudmos repo](https://github.com/akash-network/cloudmos/blob/main/deploy-web/src/utils/templates.ts#L8-L75)